femitj/Politico
# #163603955 testing all endpoint with mocha

**What does this PR do?**

Unit Testing for all endpoint with Mocha

**Description of the task to be completed**

 - Test to ensure the all endpoint are working as they should
   - POST/api/v1/auth/signup
   - POST/api/v1/auth/login
   - POST/api/v1/parties
   - GET/api/v1/parties/id
   - GET/api/v1/parties
   - POST/api/v1/offices
   - POST/api/v1/<user-id>/register
   - POST/api/v1/votes

**How should this be tested manually**

 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm run test to test endpoints.

**What are the relevant pivotal stories**
#163603955
